modules/fedimint-wallet-common/src/lib.rs

```diff
@@ -37,15 +37,24 @@ pub type PegInDescriptor = Descriptor<CompressedPublicKey>;
     Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize, UnzipConsensus, Encodable, Decodable,
 )]
 pub enum WalletConsensusItem {
-    RoundConsensus(RoundConsensusItem),
+    BlockHeight(u32), /* FIXME: use block hash instead, but needs more complicated
+                       * * verification logic */
```

Comment:
First time we used multi-line comments like this afaik, not a problem but a bit odd.